sdks/java/io/debezium/build.gradle

```diff
@@ -18,6 +18,18 @@
 import groovy.json.JsonOutput
 
 plugins { id 'org.apache.beam.module' }
+
```

Comment:
please rebase (and possibly squash commits) onto the latest master and replace these chunk to

```
applyJavaNature(
...requireJavaVersion = JavaVersion.VERSION_17
```